modules/ossm-deploy-cluster-wide-control-plane-console.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-create-smcp.adoc
+
+:_content-type: PROCEDURE
+[id="ossm-deploy-cluster-wide-control-plane-console_{context}"]
+= Configuring the control plane for cluster-wide deployment with the web console
+
+You can configure the `ServiceMeshControlPlane` resource for cluster-wide deployment using the {product-title} web console.  In this example, `istio-system` is the name of the {SMProductShortName} control plane project.
+
+.Prerequisites
+
+* The {SMProductName} Operator must be installed.
+* An account with the `cluster-admin` role.
+
+.Procedure
+
+. Log in to the {product-title} web console as a user with the `cluster-admin` role. If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.
+
+. Create a project named `istio-system`.
++
+.. Navigate to *Home* -> *Projects*.
++
+.. Click *Create Project*.
++
+.. In the *Name* field, enter `istio-system`. The `ServiceMeshControlPlane` resource must be installed in a project that is separate from your microservices and Operators.
++
+These steps use `istio-system` as an example, but you can deploy your {SMProductShortName} control plane in any project as long as it is separate from the project that contains your services.
++
+.. Click *Create*.
+
+. Navigate to *Operators* -> *Installed Operators*.
+
+. Click the {SMProductName} Operator, then click *Istio Service Mesh Control Plane*.
+
+. On the *Istio Service Mesh Control Plane* tab, click *Create ServiceMeshControlPlane*.
+
+. Click *YAML view*. The version of the {SMProductShortName} control plane determines the features available regardless of the version of the Operator.
+
+. Modify the `spec.mode` field of the YAML file to specify `ClusterWide`.
++
+.Example version {MaistraVersion} istio-installation.yaml
+[source,yaml]
+----
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: basic
+  namespace: istio-system
+spec:
+  version: v{MaistraVersion}
+  mode: ClusterWide
+----
+
+. Click *Create*. The Operator creates pods, services, and {SMProductShortName} control plane components based on your configuration parameters.
```

Comment:
We should add here that the operator also creates an SMMR by default if this does not exist with the default configuration 
```
spec:
  memberSelectors:
  - matchLabels:
      istio-injection: enabled
```
Because if the customer sets that label by default in their namespaces will be automatically added to the SMMR file

Reply:
Done